src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,146 @@
+package duke.task;
+
+import duke.storage.Storage;
+import java.util.ArrayList;
+
+public class Task {
+    protected String name;
+    protected boolean isDone;
+    protected int id;
+    private static ArrayList<Task> listOfTask = new ArrayList<>(100);
```

Comment:
Should this be called `tasks` instead of `listOfTask`? [Plural form](https://se-education.org/guides/conventions/java/intermediate.html#naming) should be used for names representing collections.